packages/docs/src/locales/i18n.ts

```diff
@@ -0,0 +1,13 @@
+import { createI18n } from 'vue-i18n'
+import { messages, DEFAULT_LANGUAGE } from './index'
+
+export const i18n = createI18n({
+  locale: DEFAULT_LANGUAGE,
+  fallbackLocale: 'en',
+  messages,
+  missing (_, key) {
```

Comment:
Can you add a comment on what this is for? Not seeing in the i18n docs.

Reply:
i18n docs:
![Screenshot from 2021-09-01 16-27-29](https://user-images.githubusercontent.com/55198465/131687134-446f9a2c-735d-4f98-8164-3b83eb549874.png)
But this code is no longer needed, because our function `$tie` handles this itself. So I will remove this from commit